tests/testthat/tests-dataset_metadata.R

```diff
@@ -22,6 +22,12 @@ test_that("check versions format", {
   ds_index  <- which(sapply(contents, function(x) x$identifier) == "FK2/HXJVJU")
   actual    <- dataset_versions(contents[[ds_index]])
 
-  expect_length(actual[[1]], 17L)
```

Comment:
This test (and a similar test below) started to fail, and blocked R CMD check from successful completion. I am unsure of the reason for this length change -- a simple addition of an existing field, or more complicated. I assume that it is not important for the existing code, but provide a more explicit test so that changes are more obvious moving forward.

Reply:
This is a nice change. I recall some attributes were added along with changes in the Dataverse API. I don't see that mattering for the performance of the R package